hedera-node/src/main/java/com/hedera/services/stats/HapiOpCounters.java

```diff
@@ -145,4 +151,12 @@ private void safeIncrement(
 			counters.get(function).getAndIncrement();
 		}
 	}
+
+	public void countDeprecatedTxnReceived() {
```

Comment:
rename the method to indicate incrementing the counter

Reply:
Added this to be similar to other increment methods in `HapiOpCounters.java`

Reply:
haha, may be we should rename all.